ban/tests/test_commands.py

```diff
@@ -84,3 +85,16 @@ def test_export_resources():
         resource['center']['coordinates'] = list(resource['center']['coordinates'])  # noqa
         assert json.loads(lines[2]) == resource
     path.unlink()
+
+
+def test_import_ignsna(staff):
+    mun_path = Path(__file__).parent / 'data/municipalities.csv'
```

Comment:
Create the municipalities with factory instead, so you don't have a dependency to "municipalities" command and its tests feature :)